src/pages/admin-panel/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export { useIsRouteActive } from "./use-is-route-active.ts";
```

Comment:
I think index file here is useless, I very rear see more than 1 hook in component